mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
```

Comment:
The callback function `deletedOrder => { ... }` should be replaced with an arrow function to maintain consistency with the rest of the codebase. It should be `deletedOrder => { ... }` instead of `.then(deletedOrder => { ... })`.